airflow/docker.go

```diff
@@ -46,7 +46,7 @@ const (
 	RuntimeImageLabel              = "io.astronomer.docker.runtime.version"
 	AirflowImageLabel              = "io.astronomer.docker.airflow.version"
 	componentName                  = "airflow"
-	podman                         = "podman"
+	podmanCmd                      = "podman"
```

Comment:
Just renamed this to be consistent with `dockerCmd`. `dockerCmd` is not simply `docker` because it conflicts with our imports of the go libs. This just makes them more similar.